src/Symfony/Component/Form/Form.php

```diff
@@ -727,11 +727,13 @@ public function isEmpty()
             }
         }
 
-        return FormUtil::isEmpty($this->modelData) ||
-            // arrays, countables
-            ((\is_array($this->modelData) || $this->modelData instanceof \Countable) && 0 === \count($this->modelData)) ||
-            // traversables that are not countable
-            ($this->modelData instanceof \Traversable && 0 === iterator_count($this->modelData));
+        if (!method_exists($this->config, 'getIsEmptyCallback')) {
+            @trigger_error(sprintf('Not implementing the getIsEmptyCallback() method in %s which implements %s is deprecated since Symfony 4.4.', \get_class($this->config), FormConfigInterface::class), E_USER_DEPRECATED);
```

Comment:
missing double quotes around `%s`

Reply:
I copied the format from existing messages (search for "Not implementing the"). Should we update those messages as well then or is this one good? 🤔

Reply:
see suggestion above
too bad for the previous messages :)
(could be another PR on 4.3)